File: TS_JT-Reader/TS_JT-Reader_Vis-Studio/TS_JT-Reader/Vec3D.ts
module C_sharp_JT_Reader.DataType {
    export class Vec3D {
        x: number , y , z;
        constructor(x: number, y: number, z: number) {
            this.x = x;
            this.y = y;
            this.z = z;
        }
        public getX(): number {
            return this.x;
        }
        public getY(): number {
            return this.y;
        }
        public getZ(): number {
            return this.z;
        }
        public getXf(): number {
            return <number>this.x;
        }
        public getYf(): number {
            return <number>this.y;
        }
        public getZf(): number {
            return <number>this.z;
        }
    }
}

File: TS_JT-Reader/C#_JT-Reader_Vis-Studio/C sharp JT-Reader/DataType/Vec3D.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace C_sharp_JT_Reader.DataType
{
    public class Vec3D
    {
        double x, y, z;

        public Vec3D(double x, double y, double z)
        {
            this.x = x;
            this.y = y;
            this.z = z;
        }

        public double getX()
        {
            return x;
        }

        public double getY()
        {
            return y;
        }

        public double getZ()
        {
            return z;
        }

        public float getXf()
        {
            return (float)x;
        }

        public float getYf()
        {
            return (float)y;
        }

        public float getZf()
        {
            return (float)z;
        }


    }
}
